src/UI/templates/default/MainControls/tpl.mode_info.html

```diff
@@ -1,5 +1,9 @@
 <div class="il-mode-info">
-	<span class="il-mode-info-content">
-		{MODE_TITLE}{CLOSE_GLYPH}
-	</span>
+	<div class="il-mode-info-content">
```

Comment:
Maybe there should be another div to contain both the content and the close button that could have the background color and the rounded corners and be pushed open by multi-line text?

Reply:
Isn't that outer container doing the fullwidth and fullheight frame around the page? I don't think it can double as the bar. Unless... maybe you could draw the page frame with il-mode-info::before/after and free up il-mode-info to be the bar this way.